pkg/kubelet/qos/policy.go

```diff
@@ -62,13 +65,44 @@ func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapa
 	// which use more than their request will have an OOM score of 1000 and will be prime
 	// targets for OOM kills.
 	// Note that this is a heuristic, it won't work if a container has many small processes.
-	memoryRequest := container.Resources.Requests.Memory().Value()
-	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+	containerMemoryRequest := container.Resources.Requests.Memory().Value()
+
+	isInPlacePodVerticalScaling := utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling)
+	if isInPlacePodVerticalScaling {
 		if cs, ok := podutil.GetContainerStatus(pod.Status.ContainerStatuses, container.Name); ok {
-			memoryRequest = cs.AllocatedResources.Memory().Value()
+			containerMemoryRequest = cs.AllocatedResources.Memory().Value()
+		}
+	}
+	oomScoreAdjust := 1000 - (1000*containerMemoryRequest)/memoryCapacity
```

Comment:
please avoid variables renaming here. It will be easier to understand that the code was not changed and will not hurt the readability

Reply:
Ideally just keep the code as-is even if it costs to re-check the feature gate value

Reply:
Done